mne/channels/data/montages/artinis-brite23.txt

```diff
@@ -0,0 +1,58 @@
+MNI Coordinates
```

Comment:
Once we are sure the elc file is correct we can delete this. Must be done before merge.

Reply:
For those playing along at home...

In MNE we use the terminology source (S) and detector (D). These artinis files use the terminology transmitter (Tx) and receiver (Rx).